src/main/java/org/alfresco/repo/security/sync/ChainingUserRegistrySynchronizer.java

```diff
@@ -714,9 +714,6 @@ public Object execute() throws Throwable
                     }
                     if (isFullSync && ChainingUserRegistrySynchronizer.logger.isWarnEnabled())
                     {
-                        ChainingUserRegistrySynchronizer.logger
-                                .warn("Full synchronization with user registry '"

```

Comment:
This does at least remove the confusing logging, which is good. But does this mean the variable name is just plain wrong? `isFullSync == true` doesn't mean that a full sync is happening?

Reply:
Yes, that's what got me confused and described in this comment https://issues.alfresco.com/jira/browse/MNT-17841?focusedCommentId=543902&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-543902. Thank you for reviewing.